config/minimalist-templates/popups_templates.yaml

```
@@ -0,0 +1,531 @@
+                            }
+                    # item2: >
+                    #   [[[
+                    #     var bri = states[entity.entity_id].attributes.brightness;
+                    #     return Math.round(bri ? (bri/256*100) : '0') + '%';
+                    #   ]]]
+                #####
+                color:
+                  type: custom:light-entity-card
+                  entity: "[[[ return entity.entity_id ]]]"
+                  persist_features: true
+                  brightness: false
+                  color_temp: false
+                  white_value: false
+                  hide_header: true
+                  effects_list: false
+                  card_mod:
+                    style:
+                      ".control":
+                        $: |
+                          #interactionLayer{ z-index: 1; filter: opacity(0.6);}
+                          #backgroundLayer{
+                            filter: blur(20px);
+                          }
+                      .: |
+                        ha-card {
+                          background: none !important;
+                          box-shadow: none !important;
+                          padding: 10px 0 10px 0 !important;
+                        }
+          item3:
+            card:
+              type: custom:button-card
+              template: popup_list_items
+              custom_fields:
+                item1:
+                  card:
+                    icon: "mdi:lightbulb-on"
+                    name: Luminosité
+                    type: "custom:button-card"
+                    template: popup_button
+                    tap_action:
+                      action: navigate
+                      navigation_path: "#brightness"
+                    state:
+                      - operator: template
+                        value: >
+                          [[[ return new URL(window.location.href).hash !== '#color' ]]]
+                        styles:
+                          icon:
+                            - color: "rgba(var(--color-blue),1)"
+                          img_cell:
+                            - background-color: "rgba(var(--color-blue), 0.2)"
+                item2:
+                  card:
+                    icon: "mdi:palette"
+                    name: Couleur
+                    type: "custom:button-card"
+                    template: popup_button
+                    tap_action:
+                      action: navigate
+                      navigation_path: "#color"
+                    state:
+                      - operator: template
+                        value: >
+                          [[[ return new URL(window.location.href).hash === '#color' ]]]
+                        styles:
+                          icon:
+                            - color: "rgba(var(--color-blue),1)"
+                          img_cell:
+                            - background-color: "rgba(var(--color-blue), 0.2)"
```

```suggestion
                            - background-color: "rgba(var(--color-blue), 0.2)"
                item3:
                  card:
                    icon: "mdi:sun-thermometer-outline"
                    name: Temperatuur
                    type: "custom:button-card"
                    template: popup_button
                    tap_action:
                      action: navigate
                      navigation_path: "#color_temp"
                    state:
                      - operator: template
                        value: >
                          [[[ return new URL(window.location.href).hash === '#color_temp' ]]]
                        styles:
                          icon:
                            - color: "rgba(var(--color-blue),1)"
                          img_cell:
                            - background-color: "rgba(var(--color-blue), 0.2)"
```

The extra button for changing the color temperature. I'm not sure about the icon for the color temperature, but i don't know what else to use.